tests/e2e/tests/showcase/mixer_showcase_test.go

```diff
@@ -35,26 +35,32 @@ func TestMixer_Report_Direct(t *testing.T) {
 	env.Configure(t,
 		test.JoinConfigs(
 			testConfig,
-			attrConfig,
+			//			attrConfig,
 			be.CreateConfigSnippet("handler1"),
 		))
 
+	dstService := env.Evaluate(t, `svc.{{.TestNamespace}}`)
```

Comment:
Hmm ... the caller would almost have to know which environment it's using to know which parameters it needs. I think this may need more thought.

Reply:
Agreed. I am happy to accommodate any recommendations here.

Reply:
We can punt on this for now.  This will all likely evolve as we keep adding functionality.